pipeline/export/nwb.py

```diff
@@ -219,7 +219,15 @@ def datajoint_to_nwb(session_key, raw_ephys=False, raw_video=False):
         unit_query = units_query & insert_key
         for unit in unit_query.fetch(as_dict=True):
             unit['id'] = max(nwbfile.units.id.data) + 1 if nwbfile.units.id.data else 0
-            unit['spike_times'] = (ephys.Unit.proj('spike_times') & unit).fetch1('spike_times')
+            aligned_times, trial_no, trial_start, trial_stop = (ephys.Unit.TrialSpikes * experiment.SessionTrial & unit).fetch('spike_times', 'trial', 'start_time', 'stop_time', order_by='trial')
+            raw_spike_times = []
+            for trial in trial_no:
+                go_cue_time = (experiment.SessionTrial * experiment.TrialEvent & session_key & {'trial': trial} & {'trial_event_type':'go'}).fetch('trial_event_time', order_by='trial')
```

Comment:
Does this line work? With `fetch()` instead of `fetch1()`, you'd get back a list